common/xrdp_constants.h

```diff
@@ -595,11 +603,16 @@
 #define  SOLIDPATTERNBRUSHONLY         0x0040
 #define  ORDERFLAGS_EXTRA_FLAGS        0x0080
 
-#define  INPUT_FLAG_SCANCODES       0x0001
-#define  INPUT_FLAG_MOUSEX          0x0004
-#define  INPUT_FLAG_FASTPATH_INPUT  0x0008
-#define  INPUT_FLAG_UNICODE         0x0010
-#define  INPUT_FLAG_FASTPATH_INPUT2 0x0020
+/* Input Capability Set (MS-RDPBCGR 2.2.7.1.6) */
+#define  INPUT_FLAG_SCANCODES          0x0001
+#define  INPUT_FLAG_MOUSEX             0x0004
+#define  INPUT_FLAG_FASTPATH_INPUT     0x0008
+#define  INPUT_FLAG_UNICODE            0x0010
+#define  INPUT_FLAG_FASTPATH_INPUT2    0x0020
+#define  INPUT_FLAG_UNUSED1            0x0040
+#define  INPUT_FLAG_UNUSED2            0x0080
+#define  TS_INPUT_FLAG_MOUSE_HWHEEL    0x0100
```

Comment:
TS_INPUT --> INPUT ?

Reply:
TS is correct. https://msdn.microsoft.com/en-us/library/cc240563.aspx

Reply:
As we already avoid the TS_ prefix we should be consistent with the that.

Reply:
I think these are from the old T128 doc.  We can add update them and add TS_ prefix at a later time.

Reply:
Okay, I'll remove TS_ prefix at least from the constants I added. BTW, lots of TS_ prefixed constants are introduced around painter.

Reply:
No, then leave it.  I'll change them all later.

Reply:
I get it.

Reply:
I'm fine with TS_ prefix, just lets change them all :)

Reply:
I added an issue for this and assigned to me.
#931